public/susi-chatbot.js

```diff
@@ -33,223 +33,203 @@ let botbuilderUserMessageTextColor = '#ffffff';
 let botbuilderBotMessageBackground = '#f8f8f8';
 let botbuilderBotMessageTextColor = '#455a64';
 let botbuilderIconColor = '';
-let botbuilderIconImg = susi_skills_deployed_url + 'customAvatars/0.png';
-
-if (typeof jQuery == 'undefined') {
-  let jqTag = document.createElement('script');
-  jqTag.type = 'text/javascript';
-  jqTag.src =
-    'https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js';
-  jqTag.onload = getTheme;
-  headTag.appendChild(jqTag);
-} else {
-  // to ensure jquery has loaded
-  getTheme();
-}
+let botbuilderIconImg = `${susi_skills_deployed_url}customAvatars/0.png`;
+
+getTheme();
```

Comment:
Use IIFE here. (https://medium.com/javascript-in-plain-english/https-medium-com-javascript-in-plain-english-stop-feeling-iffy-about-using-an-iife-7b0292aba174)